cpp/src/Slice/Parser.cpp

```diff
@@ -2146,11 +2102,6 @@ Slice::Container::validateConstant(
 {
     // isConstant indicates whether a constant or a data member (with a default value) is being defined.
 
```

Comment:
It's impossible for `type` to be `nullptr` here.
In the grammar file at the top of this PR, the `type` rule used to return `nullptr` if there's no currentContainer.
But there is always a container, so this `nullptr` return was impossible, meaning it's impossible to hit here.